test/runtests.jl

```diff
@@ -221,14 +223,43 @@ end
         neighbour_finder=DistanceNeighbourFinder(nb_matrix, 10, 1.2f0),
         thermostat=AndersenThermostat(10.0f0),
         loggers=Dict("temp" => TemperatureLogger(Float32, 10),
-                        "coords" => CoordinateLogger(Float32, 10)),
+                        "coords" => CoordinateLogger(Float32, 10),
+                        "energy" => EnergyLogger(Float32, 10)),
         timestep=timestep,
         n_steps=n_steps
     )
 
     @time simulate!(s, parallel=false)
 end
 
+@testset "General interactions" begin
+    n_atoms = 100
+    #general_inter_types = (LennardJones(true), SoftSphere(true), Mie(5, 10, true), Coulomb(true),
+    general_inter_types = (LennardJones(true), SoftSphere(true), Coulomb(true),
```

Comment:
Should we also test without neighbor lists given that there are a few atoms? Or those interactions are tested in other places already?

Reply:
Added them.